website/404.html

```diff
@@ -6,33 +6,28 @@
 
 {% block page_title_prefix %}{{_('Thunderbird')}} — {% endblock %}
 {% block page_title %}{{ _('There’s nothing here!') }}{% endblock %}
-{% block page_desc %}{{ _('The page you are looking for doesn’t exist. If you think this is a bug, you can <a href="%(bug_report_url)s">report an issue here</a>.')|format(bug_report_url=url('thunderbird.site.bug-report')) }}{% endblock %}
-
-{% block body_id %}thunderbird-404{% endblock %}
-
-{% block site_header_logo %}
-  <h2>{{ high_res_img('thunderbird/landing/wordmark.png', {'alt': _('Thunderbird'), 'width': '329', 'height': '77'}) }}</h2>
-{% endblock %}
-
-{% block page_css %}
-  <link href="/media/css/thunderbird-landing.css" rel="stylesheet" type="text/css" />
+{% block page_desc %}{{ _('The page you are looking for doesn’t exist.<br/>If you think this is a bug, you can <a href="%(bug_report_url)s" class="inline-link" target="_blank">report an issue here</a>.')|format(bug_report_url=url('thunderbird.site.bug-report')) }}{% endblock %}
```

Comment:
I really dislike the idea of having classes on <a> tags that are in translated strings.... ideally these shouldn't change unless we are changing the string. That's why the page url uses a variable like that.

Reply:
It makes sense.
I'll go around, remove those classes, and manage everything with a container class.

Reply:
It should be good now.